### DISCUSSION TITLE
Tailwind CSS

### DISCUSSION THREAD

**Discussion by LeeNPham** [vkurko/calendar#188] [General]
There are a few portions of the calendar that I want to modify styles wise. I'm not really good at regular CSS and my project uses tailwind, is there a way I can modify individual element classes inside of the calendar component so that they look different? (ex: I can currently see the month of October, when i click on it, it says it's class attribute is ec-title, but because im using tailwind, i am unsure about how I can modify that specific class so that it says something like "font-large text-3xl text-blue-300".)

My current stack uses sveltekit and tailwind css, I also updated my tailwind.config.cjs file to help me apply the styles but am pretty stuck.

**Comment by vkurko** [comment]
Please taka a look at the [theme](https://github.com/vkurko/calendar#theme) option. You can override all CSS classes that the calendar uses.

**Reply by LeeNPham** [reply]
I see what the object looks like but how do I have it apply to my component, do you have an example I could look at for svelte usage?

**Comment by LeeNPham** [comment]
@vkurko 
When I don't instantiate the theme object, the component seems to work fine.
![image](https://github.com/vkurko/calendar/assets/107651857/4e9a99dc-0fd1-4aa0-9762-b26d5e6b9d48)

but when I create it with the default values so that I can modify something like theme.title from "ec-title" to "ec-title font-bold", it bolds the thing I wanted to modify but also ruins the rest of the calendar.
![image](https://github.com/vkurko/calendar/assets/107651857/8a718e2d-51a7-4739-ae5f-3dbd162317ec)

**Reply by vkurko** [reply]
Please provide your code.
